blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,211 @@
+
+**Search:** A term like “limit 5000” is likely to bring up many hits throughout our codebase, so I used [a diff search with a timeframe](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+type:diff+limit+5000+after:%22five+months+ago%22&patternType=regexp) to scope my search to the time the behavior started appearing.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTc5?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 190px;"></iframe>
+</p>
+
+I quickly found [our search team had changed a limit](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/commit/326d250079b789ca716878c6d2e5bc694b4ea97a?visible=1) that affected one of my team’s features. I was able to understand who made the change and why they made it in a few seconds.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTgw?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+**Impact:** We could immediately respond to the customer and confirm the behavior as well as provide more context for the estimated time to fix.
+
+**Time Saved:** 18 hours waiting for a teammate to answer my question—hours that were especially important given we were actively working with a customer.
+
+### What's the Sourcegraph standard for error message syntax?
+
+I needed to finish writing the error messages that display to users for a major new feature. We have [error message guidelines](https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#error-messages) in our handbook, but it’s easier to quickly learn something with an example.
+
+**Search:** [This search](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+error+try&patternType=regexp) quickly gave me dozens of in-product examples.
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTgx?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+**Impact:** I was able to produce more unified product messaging.
+
+**Time Saved:** I saved 10 minutes I would have spent clicking through the product and avoided causing errors I would have otherwise needed to cause so I could find live examples.
+
+### Situation: Determine the different [blank]@sourcegraph emails we direct users to across our entire user-facing codebase
+
+I needed to add a contact email to some new documentation, but wasn’t sure what email addresses we’d already set up and which would be the best to reuse.
+
+**Search:** [This search](https://sourcegraph.com/search?q=context:global+r:sourcegraph/+mailto:...%40sourcegraph.com+-file:test+count:all&patternType=structural) showed me what email addresses were already in use.
```

Comment:
This says no results because it should be a structural search (even though it has type=structural)?

Reply:
Same here, the direct link works but the notebook doesn't